sdk-java/build.gradle.kts

```diff
@@ -31,7 +31,7 @@ testModuleInfo {
     requires("org.junit.jupiter.params")
     requires("org.mockito")
 
-    requiresStatic("java.annotation")
+    requires("java.annotation")
```

Comment:
```suggestion
```

This entry can and should now be removed completely the "test scope" inherits this from the "main scope" and this is now respected by the analysis. It should not suggest to add something else. I will have to check what's wrong there.